src/components/custom/timeline/timeline.tsx

```diff
@@ -0,0 +1,70 @@
+import { Timeline as DSTimeline } from "@lifesg/react-design-system/timeline";
+import * as Icons from "@lifesg/react-icons";
+import { useEffect } from "react";
+import * as Yup from "yup";
+import { useValidationConfig } from "../../../utils/hooks";
+import { Sanitize } from "../../shared";
+import { IGenericCustomElementProps } from "../types";
+import { ITimelineSchema } from "./types";
+import { Wrapper } from "../../elements/wrapper";
+
+export const Timeline = (props: IGenericCustomElementProps<ITimelineSchema>) => {
+	// =========================================================================
+	// CONST, STATE, REF
+	// =========================================================================
+	const {
+		id,
+		schema: { items, label, ...otherSchema },
+	} = props;
+	const { setFieldValidationConfig, removeFieldValidationConfig } = useValidationConfig();
+
+	// =========================================================================
+	// EFFECTS
+	// =========================================================================
+	useEffect(() => {
+		// set validation config so frontend engine's first onChange event can be fired
+		setFieldValidationConfig(id, Yup.string());
+		return () => {
+			removeFieldValidationConfig(id);
+		};
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, []);
+
+	// =========================================================================
+	// HELPER FUNCTIONS
+	// =========================================================================
+	const buildItems = () => {
+		return items.map(({ children, label, statuses, ...restOfItem }) => {
+			let itemContent: React.ReactNode;
+			if (typeof children === "string") {
+				itemContent = (
+					<p>
```

Comment:
found that without `p`, the font size will become 1rem

Reply:
oops, it's probably inheriting the styles as it's not defined by either FEE Sanitize or DS Timeline

the 18px is coming from either storybook's or the browser's default?
![Screenshot 2024-08-07 at 10 51 10 AM](https://github.com/user-attachments/assets/da379eec-d42a-435a-bf8e-9d4fe338c4a3)

so it could potentially inherit another set of styles in a different context

would propose to apply on the markup helper in the DS Timeline

Reply:
oh right, it's caused by sgds.css, I will be removing it in another PR
actually it is already rendering via `Markup` component through the `Sanitize` component
anyway I decided to DS to implement `Text.Body` for the string-based text